web_widget_x2many_2d_matrix/static/src/js/web_widget_x2many_2d_matrix.js

```diff
@@ -380,17 +390,6 @@ odoo.define('web_widget_x2many_2d_matrix.widget', function (require) {
 
         },
 
-        effective_readonly_change: function()
```

Comment:
Why this is not working?

Reply:
When testing this event was never fired; I think that is because the base `FieldX2Many` struct already catches it via:
```javascript
        this.is_loaded.done(function() {
            self.on("change:effective_readonly", self, destroy);
        });
```

Reply:
OK